xla/service/collective_permute_key.cc

```diff
@@ -34,7 +34,14 @@ std::optional<CollectivePermuteKey> GetCollectivePermuteKey(
   }
 
   const auto* cp = Cast<HloCollectivePermuteInstruction>(instruction);
-  return CollectivePermuteKey{cp->source_target_pairs(), cp->channel_id()};
+  return CollectivePermuteKey{cp->source_target_pairs(),
```

Comment:
I'm wondering if this should always be ignored. Can we try that and see if we run into test failures?